docs/custom-capo.md

```diff
@@ -0,0 +1,58 @@
+# Creating a custom cluster api provider ovirt and adding it to the cluster
+
+When we develop the CAPO its usually helpful to test it against a running cluster.
+The following doc explains how to do it.
+
+## Replacing the docker image
+
+1.Lunch a cluster:
+
+Start an Openshift cluster on top of oVirt as usual with the required release image.
```

Comment:
I would like a note here like:

For more info about installing and launching a new cluster using oVirt check:
 - [Installer Provisioned Infrastructure (IPI)](https://github.com/openshift/installer/blob/master/docs/user/ovirt/install_ipi.md)
 - [User Provisioned Infrastructure (UPI)](https://github.com/openshift/installer/blob/master/docs/user/ovirt/install_upi.md)